Documentation/network/ebpf/intro.rst

```diff
@@ -106,11 +106,9 @@ a userspace proxy (Envoy) Cilium creates the following networking objects.
   will identify candidate sockets for accelerating. These include all local node connections
   (endpoint to endpoint) and any connection to a Cilium proxy.
   These identified connections will then have all messages handled by the socket
-  send/recv hook and will be accelerated using sockmap fast redirects. The fast
```

Comment:
Pictures (`cilium/Documentation/network/ebpf/_static/cilium_bpf_*.svg`) will still refer to sockmap et al.

Reply:
Good point. I think the pictures can be updated in a separated PR though, I'll make sure to open an issue to track this when the current PR gets merged.

Reply:
https://github.com/cilium/cilium/issues/24735